assets/js/slider.js

```diff
@@ -1,7 +1,9 @@
 document.addEventListener("DOMContentLoaded", () => {
-  tns({
-    container: ".js-slider",
-    autoHeight: true,
-    speed: 400
+  [].forEach.call(document.querySelectorAll(".js-slider"), function(el) {
```

Comment:
What do you say about changing `el` to `element`?

Reply:
fine